<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRCodeGeneratorClass</class>
 <widget class="QWidget" name="QRCodeGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRCode生成器</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="textEdit">
       <property name="placeholderText">
        <string>请输入内容</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>编码类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>图像大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="widthSpinBox">
         <property name="suffix">
          <string>px</string>
         </property>
         <property name="prefix">
          <string>W: </string>
         </property>
         <property name="minimum">
          <number>24</number>
         </property>
         <property name="maximum">
          <number>2048</number>
         </property>
         <property name="singleStep">
          <number>4</number>
         </property>
         <property name="value">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="heightSpinBox">
         <property name="suffix">
          <string>px</string>
         </property>
         <property name="prefix">
          <string>H: </string>
         </property>
         <property name="minimum">
          <number>24</number>
         </property>
         <property name="maximum">
          <number>2048</number>
         </property>
         <property name="singleStep">
          <number>4</number>
         </property>
         <property name="value">
          <number>512</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="Label_2">
         <property name="text">
          <string>纠错级别</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="eccLevelComboBox">
         <property name="currentIndex">
          <number>8</number>
         </property>
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="Label_3">
         <property name="text">
          <string>四周留白</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="marginSpinBox">
         <property name="suffix">
          <string>px</string>
         </property>
         <property name="maximum">
          <number>512</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="viewWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #000;
border: 1px solid #000;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="generateBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>生成图像</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveImgBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>保存图像</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
